crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/service/MaatCourtDataService.java

```diff
@@ -15,13 +16,14 @@
 @Slf4j
 public class MaatCourtDataService {
 
-    public static final String RESPONSE_STRING = "Response from Court Data API: %s";
-    private final MaatAPIClient maatAPIClient;
+    @Qualifier("maatApiClient")
+    private final RestAPIClient maatAPIClient;
     private final ServicesConfiguration configuration;
+    public static final String RESPONSE_STRING = "Response from Court Data API: %s";
```

Comment:
Does this constant `RESPONSE_STRING` need to be `public` ? Unless it is used outside of this class probably best to reduce the scope to the bear minimum i.e. `private`
Would consider renaming to something with a bit more expression e.g. `RESPONSE_FORMAT`